src/openforms/js/components/admin/form_design/variables/registration/RegistrationsSummary.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import FAIcon from 'components/admin/FAIcon';
+import {BACKEND_OPTIONS_FORMS} from 'components/admin/form_design/registrations';
+
+const RegistrationSummary = ({name, registrationPath}) => {
```

Comment:
```suggestion
const RegistrationSummary = ({name, representation}) => {
```
reasoning for why this rename is below :)